lib/KBreadcrumbs/__tests__/KBreadcrumbs.spec.js

```diff
@@ -0,0 +1,182 @@
+import { shallowMount } from '@vue/test-utils';
+import KBreadcrumbs from '../index.vue'; // Import from index.vue
+import { renderComponent, takeSnapshot, click } from '../../../jest.conf/visual.testUtils';
+
+describe('KBreadcrumbs', () => {
+  describe('Single item breadcrumbs', () => {
+    it('should not render a single breadcrumb item by default', () => {
+      const wrapper = shallowMount(KBreadcrumbs, {
+        propsData: {
+          items: [{ text: 'Home', link: '/' }],
+        },
+      });
+      expect(wrapper.find('[data-test="breadcrumb-item"]').exists()).toBe(false);
+    });
+
+    it('should render a single breadcrumb when showSingleItem is true', () => {
+      const wrapper = shallowMount(KBreadcrumbs, {
+        propsData: {
+          items: [{ text: 'Home', link: '/' }],
+          showSingleItem: true,
+        },
+      });
+      expect(wrapper.find('[data-test="breadcrumb-item"]').exists()).toBe(true);
+    });
+  });
+
+  describe('Multiple items without overflow', () => {
+    it('should render multiple breadcrumb items when there is enough space', () => {
+      const wrapper = shallowMount(KBreadcrumbs, {
+        propsData: {
+          items: [
+            { text: 'Home', link: '/' },
+            { text: 'Category', link: '/category' },
+            { text: 'Sub-category', link: '/category/sub' },
```

Comment:
We should probably move this to a separate constant outside of the test and name it something like `MULTIPLE_BREADCRUMB_ITEMS` so that it can be reused across tests. It also makes the test as a whole more readable.